docs/installingNative.md

```diff
@@ -71,22 +71,11 @@ Now you can install the other dependencies.
 
 === "macOS"
```

Comment:
UI nit for @reteps, though it's not directly related to this PR: for tabbed interfaces like this, the theme doesn't make it easy to tell when the content controlled by the tabs ends (I know there's an `<hr>`, IMO that's not sufficient). Can we improve this? For instance, I like the styles that the [Mise docs](https://mise.jdx.dev/getting-started.html) use:

<img width="717" alt="Screenshot 2025-01-27 at 11 55 09" src="https://github.com/user-attachments/assets/2a93b397-0520-4e7f-9c77-418af05abbac" />

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/11239